pom.xml

```diff
@@ -50,107 +50,85 @@
       <version>4.13.2</version>
       <scope>provided</scope>
     </dependency>
+	<dependency>
+		<groupId>org.assertj</groupId>
+		<artifactId>assertj-core</artifactId>
+		<version>3.24.2</version>
+		<scope>test</scope>
+	</dependency>
   </dependencies>
 
-  <profiles>
-    <profile>
-      <id>release-sign-artifacts</id>
```

Comment:
Why would you remove the profile in this pull request?

Reply:
sorry, this happened accidentally during the merge. I just undo the modification.